### PR TITLE
ir_measures integration

### PR DESCRIPTION
**Pull request by cmacdonald**

### CODE REVIEW

**Comment by seanmacavaney** [comment]
looking into failing build

**Comment by cmacdonald** [comment]
i'm on it

**Comment by cmacdonald** [comment]
Yay, tests now pass. Ready for review.

File: pyterrier/pipelines.py [hunk 1]
```diff
@@ -45,21 +50,90 @@ def _color_cols(data, col_type,
     is_max[len(data) - list(reversed(data)).index(max_value) -  1] = colormaxlast_attr
     return is_max
 
+_measure_nicknames = {
```
Comment (seanmacavaney) [inline pyterrier/pipelines.py:53]: We can drop `_measure_nicknames` and `_measure_prefix` now, right?

Comment (cmacdonald) [reply pyterrier/pipelines.py:53]: Good spot. it seems likely!

**Review by seanmacavaney — APPROVED** [review]

**Pull request merged**